logprep/processor/base/rule.py

```diff
@@ -320,7 +320,7 @@ def matches(self, document: dict) -> bool:
     @classmethod
     def _create_filter_expression(cls, rule: dict) -> FilterExpression:
         special_fields = cls._get_special_fields_for_rule_matching(rule)
-        if "filter" not in rule:
+        if "filter" not in rule or rule["filter"] is None:
```

Comment:
```suggestion
        if rule.get("filter") is None:
```
How about this? 

get returns None if filter is not in rule and if filter is in rule but None

Reply:
smart :D